src/components/DropdownPanel/DropdownPanel.js

```diff
@@ -30,40 +31,29 @@ const contentIsVisible = css`
 
 const Container = styled.div`
 	position: relative;
-	width: 300px;
+	width: ${props => (props.width ? `${props.width}px` : '100%')};
```

Comment:
perhaps just leave `width` as a string and pass "px" along? just so that in other components, if we do pass width, we would allow for other units (percent, etc)